modules/ROOT/pages/index.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-With Anypoint Platform Private Cloud Edition (Anypoint Platform PCE), you run and manage Mule applications on your local servers, using your required security policies. When running Anypoint Platform PCE, MuleSoft systems do not interact with your data. You configure and run the software, and you maintain all data storage, processing, and transmission.
+With Anypoint Platform Private Cloud Edition (Anypoint Platform PCE), you run and manage Mule applications on your local servers using your required security policies. With Anypoint Platform PCE, there is no external interaction with your data. Anypoint Platform PCE control plane functionality is locally available and does not interact with runtimes over an internet connection. You configure and run the Anypoint Platform software and you maintain all data storage, processing, and transmission. Refer to the following table for components included in Anypoint Platform PCE. 
```

Comment:
I'm not sure what the following sentence means: "Anypoint Platform PCE control plane functionality is locally available and does not interact with runtimes over an internet connection". How does PCE interact with runtimes if not over the internet?